portfolio/src/main/webapp/js/script.js

```diff
@@ -102,6 +103,11 @@ document.addEventListener('DOMContentLoaded', function() {
         fullWidth: false,
         indicators: false,
     });
+    // Initialize Collapsible
```

Comment:
The code here is readable enough that these comments aren't necessary. They're mostly repeating the code. If you want to help the reader understand _why_ you put this code here, you can include that in the comments. Otherwise, feel free to remove them.

You can read more about when to add comments in go/tott-465

Reply:
Okay! Thanks for the feedback! I will fix that right away!